kolibri/plugins/coach/assets/src/state/store.js

```diff
@@ -3,28 +3,14 @@ const coreStore = require('kolibri.coreVue.vuex.store');
 const constants = require('./constants');
 
 const initialState = {
-  pageName: constants.PageNames.REPORTS,
-  pageState: {
```

Comment:
I think we're still going to need a lot of these on various subpages. Best not to delete IMO